### PR TITLE
Chore/docs v2 3

### PR DESCRIPTION
**Pull request by joshenlim**

### CODE REVIEW

File: apps/temp-docs/components/Navigation/SideBar.tsx [hunk 1]
```diff
@@ -2,21 +2,16 @@ import Link from 'next/link'
 import Image from 'next/image'
 import { useRouter } from 'next/router'
 import { IconChevronRight, IconArrowLeft } from '~/../../packages/ui'
```
Comment (saltcod) [inline apps/temp-docs/components/Navigation/SideBar.tsx:4]: can do this in another commit, but what's that thing we need to do to make it so it's just `import {thing} from 'ui'` ?

**Review by saltcod — APPROVED** [review]

File: apps/temp-docs/components/Navigation/SideBar.tsx (hunk 1 above)
Comment (joshenlim) [reply apps/temp-docs/components/Navigation/SideBar.tsx:4]: oh yeah i'll do a general clean up of these - i think its already set up to do so
